File: KHJ/programmers/220126_0_연속된수의합.java
import java.util.stream.*;

class Solution {
    public int[] solution(int num, int total) {
        int base = (total - IntStream.range(0,num).sum())/num;
        int[] answer = IntStream.range(0,num).map(i -> base + i).toArray();
        
        return answer;
    }
}

File: KHJ/programmers/220126_0_연속된수의합.py
def solution(num, total):
    div,mod = divmod(num,2)
    base = 0
    
    if mod != 0:    #홀수
        base = (total // num)  - (num-1)//2
    else:
        base = ((total - num//2) // num) - (num//2-1)
    
    answer = [base + i for i in range(num)]
    
    return answer